src/components/styled/StyledTimer.js

```diff
@@ -0,0 +1,49 @@
+import styled from 'styled-components';
+
+// upper div ( right under the Fragment)
```

Comment:
please dont leave any comments in your code. It is good code, if the code speaks for itself ;)  Try to use better naming to prevent commenting.